content/it/docs/concepts/overview/components.md

```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
 
 ### cloud-controller-manager
 
-Il [cloud-controller-manager](/docs/tasks/administer-cluster/running-cloud-controller/) esegue i controller che interagiscono con i cloud provider responsabili per la gestione dell'infrastruttura sottostante al cluster, in caso di deployment in cloud.
-Il cloud-controller-manager è una funzionalità alpha introdotta in Kubernetes 1.6.
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
 
-Il cloud-controller-manager esegue esclusivamente i cicli di controllo specifici dei cloud provider.
-È possibile disabilitare questi cicli di controllo usando il kube-controller-manager.
-È inoltre possibile disabilitare i cicli di controllo settando il parametro `--cloud-provider` con il valore `external` durante l'esecuzione del kube-controller-manager.
+Il cloud-controller-manager esegue controller specifiche del tuo cloud provider.
```

Comment:
I would say that controller is to be considered as male not female. What's your feeling on this?

Reply:
I started with "la" but I prefered to use "il" for object and I think this should be used as rule.